### ISSUE TITLE
Unifying of non-active documents doesn't work

### ISSUE THREAD

**Issue by jakubbielawa** [jakubbielawa/LineEndingsUnifier#4]
A user reported that after selecting "Unify Line Endings In This File/Folder/Project/Solution", only active documents are modified - despite what the output window says.

**Comment by jakubbielawa** [comment]
[Line Endings Unifier Debug Version.zip](https://github.com/jakubbielawa/LineEndingsUnifier/files/588204/Line.Endings.Unifier.Debug.Version.zip)

**Comment by ikafire** [comment]
Hi, I am the one who reported this issue a few weeks ago.
I found the cause of this problem - my "Save Files After Unifying" option was somehow set to false. Maybe a non-active document cannot have unsaved changes, so the unified result would be discarded automatically? After I set "Save Files After Unifying" to true, the unifier works like a charm.
Is this the intended behavior? If so then it's my bad, sorry.
